tls/extensions/s2n_npn.c

```diff
@@ -142,7 +153,7 @@ int s2n_npn_encrypted_extension_recv(struct s2n_connection *conn, struct s2n_stu
 
 const s2n_extension_type s2n_npn_encrypted_extension = {
     .iana_value = TLS_EXTENSION_NPN,
-    .is_response = true,
+    .is_response = false,
```

Comment:
Isn't this still a response? What changed?

Reply:
It turns out that you actually can't have an extension that is both a request and a response, due to this line here: https://github.com/aws/s2n-tls/blob/main/tls/extensions/s2n_extension_type.c#L170.

Reply:
Npn just has to be a rebel.

But this shouldn't be a problem. If we've gotten to even considering this extension, we've definitely received the request.